packages/fether-electron/src/main/app/methods/setupRequestListeners.js

```diff
@@ -5,9 +5,19 @@
 
 import electron from 'electron';
 
+import { CSP } from '../utils/csp';
 import messages from '../messages';
+import { TRUSTED_URLS } from '../constants';
+import Pino from '../utils/pino';
+
+const pino = Pino();
 const { ipcMain, session } = electron;
 
+pino.debug(
```

Comment:
I believe this debug should go inside the `setupRequestListeners` function, or else it'll be run on startup, not when the CSP are being configured.